source/dubregistry/web.d

```diff
@@ -139,6 +140,33 @@ class DubRegistryWebFrontend {
 		getPackageVersion(req, res, _packname, null);
 	}
 
+	@path("/packages/:packname/logo")
+	void getPackageLogo(HTTPServerRequest req, HTTPServerResponse res, string _packname)
+	{
+		bdata_t rev, logo;
+		logo = m_registry.getPackageLogo(_packname, rev);
+
+		if (logo.length) {
+			// TODO: add caching, etag (using rev), content-control, etc.
+			res.writeBody(logo, "image/png");
+		} else {
+			bool acceptsSVG;
+			// make sure requester actually supports svg, if a custom IDE or tool for fetching images is used it should only get png if it doesn't support svg.
+			// if requester is an IDE sending Accept: */*, but not accepting SVG it's their own fault.
+			foreach (accept; req.headers.get("Accept", "").splitter(",")) {
+				if (accept.startsWith("image/*", "image/svg", "*/*", "*/svg")) {
```

Comment:
Isn't this `among`?

Reply:
`image/*; q=0.8`

Reply:
eventually gonna make a PR to vibe.d for proper Accept parsing